internal/config/config.go

```diff
@@ -67,6 +68,16 @@ type Egress struct {
 	// file, can only be set through the environment variable.
 	SnykServiceAccountToken string `json:"-" env:"SNYK_SERVICE_ACCOUNT_TOKEN"`
 }
+
+type Route struct {
+	// OrganizationID is the snyk organization ID where data should be routed to.
+	OrganizationID string `json:"organizationID"`
+	// Namespaces which are correlated with given OrganizationID
+	// Each route has to contain at least one namespace
+	// Supports * to match all namespaces
+	Namespaces []string `json:"namespaces"`
```

Comment:
I'm generally wondering how to handle cluster-scoped resources. Add a separate field to enable / disable these? 
If we don't add a separate field, this config would send all cluster-scoped resources + from the default namespace to a specific org:
```yaml
routes:
- organizationID: blabla
  namespaces: ["default"]
- organizationID: blabla
  // allow namespaces to be nil to indicate cluster-scoped resources. But do not treat `[]` the same!
```

which is also a bit weird.
we could also use an empty namespace (`""`) to indicate cluster-scoped resources, but that's also a bit weird. 😄  

So maybe:
```yaml
routes:
- organizationID: blabla
  clusterScopedResources: true
  namespaces: ["default"]
```

and treat `namespaces == nil` / `len(namespaces) == 0` the same?

Reply:
I think this is the most clear one , good catch with cluster resources 👍 
```
routes:
- organizationID: blabla
  clusterScopedResources: true
  namespaces: ["default"]
```